src/SettingsServiceProvider.php

```diff
@@ -40,7 +40,7 @@ public function boot()
         $this->loadTranslationsFrom(realpath(__DIR__.'/resources/lang'), 'backpack');
 
         // publish the migrations and seeds
-        $this->publishes([__DIR__.'/database/migrations/' => database_path('migrations')], 'migrations');
+        $this->loadMigrationsFrom(__DIR__.'/database/migrations/');
```

Comment:
That's why, while we're still supporting 5.3, I think it's safe to just publish the migrations. L5.4 will still run them (because they're published), it would just throw a warning. Right, @xxdalexx ?